.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
**[Optional]**

- [ ] Your commit messages are not capitalize :fearful: Kindly capitalize them to make your pull request professional :+1: